src/components/Profile/Media.js

```diff
@@ -1,7 +1,8 @@
 import React from "react";
 import { Grid, Typography, Fab } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
-import CreateTweet from "../HomePage/CreateTweet/CreateTweet";
+import CreateTweet from "./CreateTweet/CreateTweet";
```

Comment:
Directory path really looks odd.
You should have something like **./CreateTweet**